lib/siwapp/commons/tax.ex

```diff
@@ -13,7 +13,11 @@ defmodule Siwapp.Commons.Tax do
     field :enabled, :boolean, default: true
     field :default, :boolean, default: false
     field :deleted_at, :utc_datetime
-    many_to_many :items, Item, join_through: "items_taxes"
+
+    many_to_many :items, Item,
+      join_through: "items_taxes",
+      join_keys: [taxes_id: :id, items_id: :id],
+      on_replace: :delete
```

Comment:
Explain why do you add these 2 options.

Reply:
To do the association correctly between items and taxes by items_taxes. It does not allowed me to do that association unless I add the join_keys. The on_replace option is added in order to delete a tax associated to an item. In case we need to delete, of course.